src/mopsy/helpers.py

```diff
@@ -117,3 +117,19 @@ def apply(
     """
     tmat = get_matrix_type(mat)
     return tmat.apply(func, group=group, axis=axis)
+
+
+def fapply(
+    mat: Union[numpy.ndarray, scipy.sparse.spmatrix], funcs: list, axis: int,
+):
+    """a reduction apply with multiple functions
+
+    Args:
+        mat (Union[numpy.ndarray, scipy.sparse.spmatrix]): matrix
+        funcs (list): functions to be called.
```

Comment:
missing def for axis and group

Reply:
added

Reply:
use the same format from the `apply` function

Reply:
fixed format